app/views/api/v2/locations/create.json.rabl

```diff
@@ -0,0 +1,3 @@
+object @taxonomy
+
+extends "api/v2/taxonomies/create"
```

Comment:
Good catch.  Adding the rabl templates is the way we chose to do it in d575926a, so this is best for consistency.